supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -21,3 +21,72 @@ When the format _day Month year_ causes a presentation or clarity issue, use ins
 
 [[product-names-versions-ref]]
 == Product names and version references
+
+////
+PROPOSED ADDITIONS?
+master-docinfo.xml specific and portal-specific version variables
+URL fragments and composed links?
+correct sequence when combining attributes
+ensure proper sequence for compound attributes
+syntax recommendations: what separators to use? dashes/hyphens, underscores?
```

Comment:
Do mod docs guidelines provide guidance on this? @sterobin ?

Reply:
I see that inoxx03 has removed this content. So, in order of moving this PR along, I'm resolving this conversation.